src/NuGet.Protocol.VisualStudio/UISearchResourceV2.cs

```diff
@@ -43,6 +44,13 @@ public UISearchResourceV2(IPackageRepository repo)
         {
             return await Task.Run(() =>
             {
+                // Check if source is available.
```

Comment:
The comment reads as applying only to the if. You might want to clarify.

I'm ok either way 

-----Original Message-----
From: "Harsh Gupta" <notifications@github.com>
Sent: ‎6/‎11/‎2015 5:27 PM
To: "NuGet/NuGet3" <NuGet3@noreply.github.com>
Cc: "Yishai Galatzer" <yishai_galatzer@yahoo.com>
Subject: Re: [NuGet3] Making the package manager detect and report errors for source paths … (#49)

In src/NuGet.Protocol.VisualStudio/UISearchResourceV2.cs:
> @@ -43,6 +44,13 @@ public UISearchResourceV2(IPackageRepository repo)
>          {
>              return await Task.Run(() =>
>              {
> +                // Check if source is available.
the source might be valid, but not available (network issues etc), .. if you think valid is better ill change that.
—
Reply to this email directly or view it on GitHub.